doc/api/next_api_changes/2019-09-04-AL.rst

```diff
@@ -0,0 +1,10 @@
+Matplotlib now uses Pillow to save and read pngs
```

Comment:
Please adapt to the new way of documentation API changes.

Reply:
done